#! /usr/bin/env python
# Task Long configuration

PACKAGE='floor_nav'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *
from task_manager_lib.parameter_generator import *

gen = TaskParameterGenerator()
#       Name                    Type            Description
#       Default    Min    Max
gen.add("target",     double_t, 0,    "Target angle",  0.)
gen.add("relative",     bool_t, 0,    "When true, the target angle is relative to the current position",  False)
gen.add("k_theta",        double_t, 0,    "Gain for angular control of the robot angle",  0.001)
gen.add("max_angular_velocity",   double_t, 0,    "Max allowed velocity",  2.0)
gen.add("angle_threshold",double_t, 0,    "Angle at which the target is considered reached",  10.0)
gen.add("img_target",double_t, 0,    "Angle at which the target is considered reached",  128.0)


exit(gen.generate(PACKAGE, "floor_nav", "TaskStareAtFace"))

